### ISSUE TITLE
Hebb's Rule

### ISSUE THREAD

**Issue by dlee138** [openconnectome/Statistical-Connectomics-Class#194]
Although Hebb's Rule summarizes that neurons that fire together wire together, the "out of sync lose their link" statement is often ignored.  Are there any papers (like the Hebbian phase sequence paper), that study memory, but investigate "forgetting" rather than learning?

**Comment by aceeccc** [comment]
There are papers on short-term learning...Oh! I finally found something that had the word "unlearning" and after searching that in pubmed a bunch of things came up. They might not all mention Hebbian plasticity, but they're focused on "forgetting."
Neuronal regulation versus synaptic unlearning in memory maintenance mechanisms.
http://www.ncbi.nlm.nih.gov/pubmed/10221581
The spacing principle for unlearning abnormal neuronal synchrony.
http://www.ncbi.nlm.nih.gov/pubmed/25714553
This second one is about reversing learned signalling from psychiatric disorders.

**Comment by ElanHR** [comment]
There are a number of papers that use a Hebbian approach to investigate the role of of timing as it relates to forgetting. I believe the relevant keyword is "Spike-timing-dependent Plasticity" 

Spike-timing-dependent plasticity and relevant mutual information maximization.
http://www.ncbi.nlm.nih.gov/pubmed/12816563

Spike-timing-dependent Plasticity
http://en.wikipedia.org/wiki/Spike-timing-dependent_plasticity

**Comment by yaxigeigei** [comment]
Let's take a step back by thinking about what is the relationship between learning/memory and synaptic plasticity (or in any generalized fancy form like spike-timing-dependent or -independent synaptic plasticity with homeostatic regulation)? In fact, wiring together doesn't necessary mean learning or memorizing. In the case like auditory fear conditioning (mediated by synapses in amygdala) it does (1), while adding strengthened synapses destabilizes the hippocampus network and induces forgetting (2). On the other hand, motor learning requires weakening of "unwanted" activation (3) in addition to strengthen some. 
So, it is the network that encodes information and provides input-output functions. Either memorizing or forgetting can be result of both wiring and losing connections. Population recoding now really provides us a chance to investigate the structure of neural networks and understand how synaptic plasticity is regulated in a circuit level in order to achieve certain function. 

1) Hebbian and neuromodulatory mechanisms interact to trigger associative memory formation, 2014
2) Hippocampal Neurogenesis Regulates Forgetting During Adulthood and Infancy, 2014
3) Cortical activity in the null space: permitting preparation without movement, 2014

**Comment by whock** [comment]
I agree with Duo (@yaxigeigei) that the situation is more complex than the wiring diagram. But I believe Hebb's rule as commonly expressed in Dr. Schatz's phrase "fire together, wire together" is more nuanced than simply saying that wiring together means learning. At its core, Hebbian plasticity is a bi-directional relationship between a structure (the net) and a pattern (spikes) that alter one another over time. The interplay between the two is key here and is the real impetus for learning (that is, if the interplay has the right structure, cf STDP). The structure constrains what type of activity can propagate through it, while the pattern of activity alters the structure itself via diverse learning rules. So Hebb isn't really saying that the wiring diagram is learning. He, and those after him, are more commenting on a dynamic relationship, a sort of dance, between the wiring and the activity. 

One of the papers you cite makes this point well (1). In it the authors find that neurogenesis (new neurons being born and integrated into a circuit) within the hippocampal dentate gyrus (DG) actually disrupts memory recall. But the reason why the addition of more wiring caused forgetting was because these new cells competed for inputs/outputs with existing cells. They disrupted the carefully constructed network architecture that allowed memory patterns to be expressed. This result highlights the point above - it is the tight relation between wiring and firing that allows memory encoding / retrieval. When they become uncoupled, as in the paper mentioned, it's like the proverbial bull in a china shop. So learning and memory isn't really about the wiring at all but rather a relationship between the wiring and the firing.

(1) Hippocampal Neurogenesis Regulates Forgetting During Adulthood and Infancy, 2014
